src/main/java/acme/entities/courseLectures/CourseLecture.java

```diff
@@ -0,0 +1,34 @@
+
+package acme.entities.courseLectures;
+
+import javax.persistence.ManyToOne;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
```

Comment:
Añadir importaciones necesarias
```suggestion
import lombok.Getter;
import lombok.Setter;
import javax.persistence.Entity;
```